### DISCUSSION TITLE
Потоки ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#43] [Announcements]
# Потоки ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/api/stream/

**Comment by masiacra** [comment]
Откуда взят оригинал?
Просто местами ужасный перевод

**Reply by bndby** [reply]
это старый перевод, новый уже в пути

**Reply by masiacra** [reply]
Буду ждать. А то смущает эти звонки вместо вызовов в тексте и вилки вместо fork)

**Comment by kukuruku62** [comment]
не поверите, но это все взято с офсайта ноды, но там все еще хуже.
